ang/test/civiawards/award-creation/directives/review-panels/review-panels.directive.spec.js

```diff
@@ -271,58 +266,138 @@
+            contact_settings: {
+              exclude_groups: [],
+              include_groups: [],
+              relationship: []
+            },
+            visibility_settings: {
+              application_status: [],
+              anonymize_application: '0'
+            },
+            is_active: '0',
+            formattedContactSettings: {
+              include: [],
+              exclude: [],
+              relation: []
+            },
+            formattedVisibilitySettings: {
+              applicationStatus: []
+            }
+          }]);
+        });
+
+        it('shows the contacts name in the contact list', () => {
+          expect(crmApi).toHaveBeenCalledWith('Contact', 'get', {
+            sequential: 1,
+            return: ['display_name'],
+            id: { IN: ['4', '2', '3', '1'] },
+            options: { limit: 0 }
+          });
         });
       });
 
-      it('displays all existing review panels for the award', () => {
-        expect($scope.existingReviewPanels).toEqual([{
-          id: '46',
-          title: 'New Panel',
-          case_type_id: '62',
-          contact_settings: {
-            exclude_groups: ['1'],
-            include_groups: ['2'],
-            relationship: [{
-              is_a_to_b: '1',
-              relationship_type_id: '14',
-              contact_id: ['4', '2']
-            }, {
-              is_a_to_b: '0',
-              relationship_type_id: '14',
-              contact_id: ['3', '1']
-            }]
-          },
-          visibility_settings: {
-            application_status: ['1'],
-            anonymize_application: '1'
-          },
-          is_active: '1',
-          formattedContactSettings: {
-            include: ['Group 2'],
-            exclude: ['Group 1'],
-            relation: [{
-              relationshipLabel: 'Benefits Specialist is',
-              contacts: ['Shauna Barkley', 'Shauna Wattson']
-            }, {
-              relationshipLabel: 'Benefits Specialist',
-              contacts: ['Kiara Jones', 'Default Organization']
-            }]
-          },
-          formattedVisibilitySettings: {
-            applicationStatus: ['Ongoing']
-          }
-        }]);
+      describe('when there no relationships in all review panels', () => {
+        beforeEach(() => {
+          setApiActionHandlers({ noRelationPresentInAllReviewPanels: true });
+          $scope.awardId = '10';
+          createController();
+          $scope.$digest();
+        });
+
+        it('fetches all exisiting review panels for the award', () => {
+          expect(crmApi).toHaveBeenCalledWith('AwardReviewPanel', 'get', {
+            sequential: 1,
+            case_type_id: '10',
+            options: { limit: 0 }
+          });
+        });
+
+        it('displays all existing review panels for the award', () => {
+          expect($scope.existingReviewPanels).toEqual([{
+            id: '47',
+            title: 'New Panel 2',
```

Comment:
Same like the comment above. Do you think we should move it to `mocks/data` folder ?

Reply:
Moved to a new variable, as discussed.